pydoctor/templatewriter/pages/__init__.py

```diff
@@ -445,32 +467,33 @@ def baseTables(self, request, item):
                                                loader))
                 for b, attrs in baselists]
 
-    def baseName(self, data: Tuple[model.Class, ...]) -> List[str]:
+    def baseName(self, data: Sequence[model.Class]) -> "Flattenable":
         page_url = self.page_url
-        r = []
+        r: List["Flattenable"] = []
         source_base = data[0]
         r.append(tags.code(epydoc2stan.taglink(source_base, page_url, source_base.name)))
         bases_to_mention = data[1:-1]
         if bases_to_mention:
-            tail = []
+            tail: List["Flattenable"] = []
             for b in reversed(bases_to_mention):
                 tail.append(tags.code(epydoc2stan.taglink(b, page_url, b.name)))
                 tail.append(', ')
             del tail[-1]
             r.extend([' (via ', tail, ')'])
         return r
 
-    def functionExtras(self, ob):
+    def functionExtras(self, data: model.Documentable) -> List["Flattenable"]:
```

Comment:
We name the documentable `obj` everywhere, why do you change that here ?

Reply:
It was named `data` in `CommonPage` and I copy-pasted the annotated header. I could rename it in `CommonPage` instead.